packages/explat-client/src/internal/experiment-assignments.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Internal dependencies
```

Comment:
It took me a while to understand that `Internal dependencies` references to the imports, not the entire file. In general, the first block comment usually describes the file, so this is pretty confusing. How about we change these to be more descriptive and make the internal/external dependencies one-line comments?

Taking this file as an example, it'd be:

```javascript
/**
 * Functions to manipulate and check ExperimentAssignment objects.
 */

// Internal dependencies
import type { ExperimentAssignment } from '../types';
...
```

Reply:
For what it's worth, this is a convention enforced by our ESLint rules and is followed by both `wp-calypso` and WordPress/Gutenberg.

Reply:
Yep, they are compulsory if we are keeping with the conventions. I do think the files are self-explanatory enough and are pretty simple, plus we will have a little file guide in the readme.

Reply:
Makes sense.